sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/src/test/java/com/azure/messaging/eventhubs/checkpointstore/blob/BlobCheckpointStoreTests.java

```diff
@@ -74,166 +84,270 @@ public void afterEach() throws Exception {
         Mockito.framework().clearInlineMock(this);
     }
 
+    /**
+     * Tests that listing ownership works.
+     */
     @Test
-    public void testListOwnerShip() {
+    public void testListOwnership() {
+        final String fullyQualifiedNamespace = "namespace.microsoft.com";
+        final String eventHubName = "MyEventHubName";
+        final String consumerGroup = "$Default";
+        final String prefix = getLegacyPrefix(fullyQualifiedNamespace, eventHubName, consumerGroup);
+        final String ownershipPrefix = prefix + "/ownership/";
```

Comment:
nit; Should we use the const OWNERSHIP_PATH here? (to align with the use of the const CHECKPOINT_PATH in `testListCheckpoint()`)